source/NVDAObjects/__init__.py

```diff
@@ -558,12 +558,36 @@ def _get_rowNumber(self):
 		"""
 		raise NotImplementedError
 
+	def _get_presentationalRowNumber(self):
+		"""
+		An optional version of the rowNumber property used purely for speech and braille presentation if implemented.

```

Comment:
Some of the lines in here seem to have a pretty long length. I recall @feerrenrut preferring a length of at most 120, preferably around 80 characters.